### ISSUE TITLE
Normal: change default parameter from `var` to `sd`

### ISSUE THREAD

**Issue by kevindavisross** [dlsun/symbulate#55]

**Comment by hayate0304** [comment]
Done. I switched the `sd` and `var` arguments order. If `var` is provided, the distribution takes the `var` instead of the `sd` >> maybe a little bit confused. I can remove the the var argument if you want.
`    def __init__(self, mean=0.0, sd=1.0, var=None):`
![image](https://user-images.githubusercontent.com/23329938/43813798-1a99efc4-9a7c-11e8-817b-4850f021f48d.png)

**Comment by kevindavisross** [comment]
No, don't remove `var`.  But do raise an exception when the user provides both `var` and `sd` (we should have done this before), something like 'At most one of sd and var can be specified.'
